internal/spec/spec.go

```diff
@@ -236,6 +245,9 @@ func (r *RunnerSpec) MergeExtraSpecs(extraSpecs *extraSpecs) {
 	if extraSpecs.EnableBootDebug != nil {
 		r.EnableBootDebug = *extraSpecs.EnableBootDebug
 	}
+	if extraSpecs.DisableUpdates != nil { // Access promoted field from embedded CloudConfigSpec
```

Comment:
`DisableUpdates` is not really inside `CloudConfigSpec` as the comment suggests. But it is indeed a flag we can set inside `UserDataOptions`, which is inside the bootstrap params. The comment is a bit misleading. I think it can be removed.